cloudcix_primitives/templates/firewallns/nftables.conf.j2

```diff
@@ -67,57 +77,77 @@ table inet {{ table }} {
     }
 {% endfor %}
 {% endif %}
-{# ----------------------------------------------------------------------- #}
-{% if prerouting_rules|length > 0 %}
-    # Pre-Routing, Inbount NAT (DNAT), Only IPv4
-    chain prerouting {
-        type nat hook prerouting priority {{ priority }}; policy accept;
+{# ---------------------NAT Pre Routing---------------------------------- #}
+{% if nat_prerouting is not None and nat_prerouting['lines']|length > 0 %}
+    # Inbound NAT (DNAT), Only IPv4 at Pre Routing stage,
+    # for nat type chains default policy is accept
+    chain nat_prerouting {
+        type nat hook prerouting priority {{ nat_prerouting['priority'] }}; policy accept;
         # Inbound destination NAT 1:1
-{% for rule in prerouting_rules %}
+{% for line in nat_prerouting['lines'] %}
+        {{ line }}
+{% endfor %}
+    }
+{% endif %}
+{# ---------------------NAT Post Routing--------------------------------- #}
+{% if nat_postrouting is not None and nat_postrouting['lines']|length > 0 %}
+    # Outbound NAT(SNAT), Only IPv4 at Post-Routing stage,
+    # for nat type chains default policy is accept
+    chain nat_postrouting {
+        type nat hook postrouting priority {{ nat_postrouting['priority'] }}; policy accept;
+        # Source NAT 1:N
+    {% for line in nat_postrouting['lines'] %}
```

Comment:
https://github.com/CloudCIX/primitives/blob/main/cloudcix_primitives/templates/net/configs/netplan.yaml.j2

During the development of this template, we discovered that the jinja for and if statements should not be indented as it skews the formatting of what is rendered. 

Can you apply these lessons learned here?